project/gen2.scala

```diff
@@ -120,6 +120,12 @@ object gen2 {
         io.transact(xa).unsafePerformIO
       },
 
+      enum("F2WindowCover") {
+        type F2WindowCoverRec = Record.`'tag -> String, 'shortName -> String, 'longName -> String, 'tccValue -> String`.T
+        val io = sql"select id, id tag, short_name, long_name, tcc_value from e_f2_window_cover".query[(String, F2WindowCoverRec)].list
+        io.transact(xa).unsafePerformIO
+      },
```

Comment:
Note, the `WindowCover` parameter is actually generated source, like all the other instrument fields.  This is actually part of the SBT build.  I would be interested in an explanation of what is going on here with `Record` and `.T` etc.  I just followed the pattern like a monkey and it worked.

Unfortunately whenever you touch this file you have to `reload` and recompile to create the enumeration.  That causes Idea to forget that you marked `target/src_managed` as generated source.  Maybe we can change where `src_managed` is written and make this a little easier.